docs/vdpa/README.md

```diff
@@ -0,0 +1,68 @@
+# Using vDPA devices in Kubernetes
+## Introduction to vDPA
+vDPA (Virtio DataPath Acceleration) is a technology that enables the acceleration
+of virtIO devices while allowing the implementations of such devices
+(e.g: NIC vendors) to use their own control plane.
+
+The consumers of the virtIO devices (VMs or containers) interact with the devices
+using the standard virtIO datapath and virtio-compatible control paths (virtIO, vhost).
+While the data-plane is mapped directly to the accelerator device, the contol-plane
+gets translated the vDPA kernel framework.
+
+The vDPA kernel framework is composed of a vdpa bus (/sys/bus/vdpa), vdpa devices
+(/sys/bus/vdpa/devices) and vdpa drivers (/sys/bus/vdpa/drivers).
+Currently, two vdpa drivers are implemented:
```

Comment:
```suggestion
The vDPA kernel framework is composed of a vDPA bus (/sys/bus/vdpa), vDPA devices
(/sys/bus/vdpa/devices) and vDPA drivers (/sys/bus/vdpa/drivers).
Currently, two vDPA drivers are implemented:
```
Not sure if the lower case was on purpose or not, if not, then to be consistent update to "vDPA". Just consider this a global comment. There are several instances of lower case "vdpa" below.